lib/middleman-cdn/cdns/cloudflare.rb

```diff
@@ -23,6 +23,7 @@ def self.example_configuration_elements
       end
 
       def invalidate(options, files, all: false)
+        options[:invalidate_zone_for_many_files] = true unless options.has_key?(:invalidate_zone_for_many_files)
```

Comment:
Yeah it's not deprecated in the true sense of the word, I think it's just not favoured because the preferred way of naming methods in ruby has been agreed to leave out `is` and `has` as function prefixes.